lib/lightning/invocation.ex

```diff
@@ -404,13 +405,23 @@ defmodule Lightning.Invocation do
             select: %{id: wf.id, name: wf.name, project_id: wf.project_id}
           ),
         attempts: ^attempts_query
-      ]
+      ],
+      select: %{
+        id: wo.id,
+        last_finished_at: r.finished_at,
+        work_order: wo
+      }
     )
   end
 
   def list_work_orders_for_project(%Project{} = project, filter, params) do
     list_work_orders_for_project_query(project, filter)
     |> Repo.paginate(params)
+    |> find_uniq_wo()
+  end
+
+  def find_uniq_wo(page) do
+    %{page | entries: Enum.uniq_by(page.entries, fn wo -> wo.id end)}
```

Comment:
@taylordowns2000 is it possible to uniq these via the query and not in memory? I'm cautious that a uniq after the query would result in less than the page limit specifies - assuming it's possible to have more than one unique result in the first place..